apps/progress/progress.c

```diff
@@ -0,0 +1,359 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include "progress.h"
+
+void insert(struct failure *f)
+{
```

Comment:
Can you put opening curlies on the previous line?

Reply:
Not a fan of K&R's style eh? ;)